src/pages/layer-2.tsx

```diff
@@ -773,7 +773,9 @@ const Layer2Page = ({ data }: PageProps<Queries.Layer2PageQuery>) => {
                   tokenLists={l2.tokenLists}
                 >
                   {l2.purpose.map((purpose, index) => (
-                    <Pill key={index}>{purpose}</Pill>
+                    <Badge key={index} me={2}>
+                      {purpose}
+                    </Badge>
```

Comment:
These badges for some reason don't have a background color.
![image](https://user-images.githubusercontent.com/468158/212393991-3a5e06af-40ab-4b1d-85ae-1fe74e900b4e.png)

Should the default variant be transparent? I'd expect for this iteration to have the same default color that the Chakra Badge uses.